frontend/packages/console-dynamic-plugin-sdk/.gitignore

```diff
@@ -1 +1 @@
-schema
+/schema
```

Comment:
@rawagner FYI - this fixes an issue where Git would ignore `schema` directory at any level under the SDK package.